assets/src/components/apps/app/runbooks/runbook/RunbookStatus.tsx

```diff
@@ -1,18 +1,110 @@
-import { Chip, WarningIcon } from '@pluralsh/design-system'
-import { Span } from 'honorable'
+import { Chip, Tooltip, WarningIcon } from '@pluralsh/design-system'
+import { Maybe, Runbook, RunbookAlertStatus } from 'generated/graphql'
+import { Div, Flex, Span } from 'honorable'
+import moment from 'moment'
+import { useTheme } from 'styled-components'
 
 import { hasAlerts } from '../misc'
 
+function AlertsTooltipLabel({
+  alerts,
+}: {
+  alerts: Maybe<Maybe<RunbookAlertStatus>[]>
+}) {
+  const theme = useTheme()
+
+  if (!alerts || alerts.length === 0) {
+    return null
+  }
+  const DATE_PATTERN = 'MMM D, YYYY h:mm a'
+
+  return (
+    <Flex
+      whiteSpace="nowrap"
+      flexDirection="column"
+    >
+      {alerts?.map((alert, i) => {
+        const dateTime = moment(alert?.startsAt).format(DATE_PATTERN)
+
+        return (
+          alert && (
+            <Flex
+              padding="medium"
+              gap="small"
+              size={16}
+              {...{
+                ':not(:last-child)': {
+                  borderBottom: theme.borders['fill-two'],
+                },
+              }}
+            >
+              <Div paddingTop={2}>
+                <WarningIcon color={theme.colors['icon-warning']} />
+              </Div>
+              <Flex
+                key={i}
+                flexDirection="column"
+                flexGrow={1}
+                gap="xxsmall"
+                className="ttklink"
```

Comment:
Leftover?

Reply:
@dogmar I'll merge we can remove it later.

Reply:
It will be removed in my next PR.